src/pretix/control/templates/pretixcontrol/items/category.html

```diff
@@ -16,8 +16,18 @@ <h1>{% trans "Product category" %}</h1>
                         {% bootstrap_field form.internal_name layout="control" %}
                     </div>
                     {% bootstrap_field form.description layout="control" %}
-                    {% bootstrap_field form.category_type layout="control" horizontal_field_class="big-radio-wrapper col-lg-9" %}
-                    {% bootstrap_field form.cross_selling_condition layout="control" horizontal_field_class="col-lg-9" %}
+                    {% bootstrap_field form.category_type layout="control" horizontal_field_class="big-radio-wrapper col-md-9" %}
+                    <div class="row" data-display-dependency="#id_category_type_2">
+                        <div class="col-md-offset-3 col-md-9">
+                            <div class="alert alert-info">
+                                {% blocktrans trimmed %}
+                                Please note that cross-selling categories are only intended as a marketing feature and are not
+                                suitable for ensuring that products are only available in certain combinations.
```

Comment:
```suggestion
                                Please note that cross-selling categories are intended as a marketing feature and are not
                                suitable for strictly ensuring that products are only available in certain combinations.
```
Marketing is not "only", maybe this makes it more clear?